src/main/java/net/sf/rails/ui/swing/GameSetupController.java

```diff
@@ -183,25 +189,44 @@ private void startNewGame() {
         }
     }
 
+    private boolean isOurs(File f) {
+        String ext = StringUtils.substringAfterLast(f.getName(), ".");
+        if ( StringUtils.isBlank(ext) ) {
+            // ignore files with no extensions
+            return false;
+        }
+        switch (ext) {
+            case GameUIManager.DEFAULT_SAVE_EXTENSION:
+            case GameUIManager.DEFAULT_SAVE_POLLING_EXTENSION:
+                return true;
+            default:
+                return ext.equals(savedFileExtension);
+        }
+    }
```

Comment:
I can't identify what this method does by reading the name. Is there maybe a more fitting method name? Alternatively we should add a comment to the method.

Reply:
Method name + argument is defining-- is Our file?

Reply:
Yes but how do you define `our file`?
If you mean files belonging to Rails then wouldn't `belongsToRails(...)` be more appropriate?